aut/commands/info.py

```diff
@@ -0,0 +1,74 @@
+"""
+The "info" command group
+"""
+
+
+from click import group, command, echo
+
+
+# Disable pylint warning about imports outside top-level.  We do this
+# intentionally to try and keep startup times of the CLI low.
+
+# pylint: disable=import-outside-toplevel
+
+
+@group(name="info")
+def info_group() -> None:
+    """
+    Commands for querying info information.
```

Comment:
From a UX point of view, I think it a more intuitive tool is where all the primary commands map onto distinct objects, such as: `account`, `contract`, `validator`, etc...

On reflection I feel that `info` as a primary command has two disadvantages: (1) by itself is not a self-descriptive name, as `info` is generic and could contain anything at all, and related is (2) it can introduce doubt in everyday interactions because if I need info about some _object_, I cannot be sure if it will be under _info_ or some other primary command.

A sign of an issue may be the description of the command:

>   Commands for querying info information.

I cannot guess what I can do with `info` until I type `aut info --help` and see what's available, and so the description is not very useful.

More specifically, I can see that the newly added commands relate to retrieving information about the autonity contract version used. This seems closer to a property of the `protocol`, do you think that makes sense?